src/clients/python/client_build.py

```diff
@@ -0,0 +1,17 @@
+from cffi import FFI
```

Comment:
I think we should generate a modified `tb_client.h` in zig instead of hacking the existing one in phyton.